src/aspire/source/simulation.py

```diff
@@ -43,6 +49,20 @@ def __init__(self, L=8, n=1024, states=None, filters=None, offsets=None, amplitu
         self.vols = self._gaussian_blob_vols(L=self.L, C=self.C, seed=seed)
         self.seed = seed
 
+        # We have all information to be able to initialize the model pipeline.
+        self.init_model_pipeline()
+
+        # The generation pipeline of a Simulation uses the same transforms as the model pipeline,
+        # except that we might add a NoiseAdder transform at the end
+        self.generation_pipeline = Pipeline(self.model_pipeline.xforms, memory=self.model_pipeline.memory)
+
+        self.noisy = noisy
```

Comment:
Isn't it enough just to set the noisy variance to zero to disable the noise?

Reply:
There is currently no `noise_variance` parameter when creating a `Simulation`, just a `noise_filter`. But this is a good idea - I've replace `noisy` (boolean) with a `noise_variance` parameter (float), with a default value of 1.0. A value of 0.0 will be equivalent to no noise.

Reply:
Well it's good to have `noise_filter` specify the noise in the `Simulation`. Having `noise_variance` as well is a bit redundant. However, a `noise_filter` equal to something like `ScalarFilter(0)` (which is maybe its own class, `ZeroFilter()`, in analogy to `IdentityFilter()`), would be equivalent.

Reply:
Remove the `noise_variance` parameter from Simulation (just rely on `noise_filter`)

Reply:
Added a `ZeroFilter` class. Removed a `noise_variance` parameter from `Simulation` constructor.